osutil/disks/disks_linux.go

```diff
@@ -374,6 +375,13 @@ func (d *disk) UsableSectorsEnd() (uint64, error) {
 		return byteSz / sectorSz, nil
 	}
 
+	calculated, err := CalculateLastUsableLBA(d.devname)
```

Comment:
we need a unit test in disks_linux_test for the case where this is successful to demonstrate we don't try sfdisk in the case where this works, currently we are only testing the case where this fails and we fall back to using sfdisk

Reply:
Done.